receiver/huaweicloudcesreceiver/internal/ces_client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/huaweicloudcesreceiver/internal"
+
+import (
+	ces "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/ces/v1"
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/ces/v1/model"
+)
+
+//go:generate mockery --name CesClient --case=underscore --output=../mocks
```

Comment:
i think the `mocks` folder should be under `internal` we don't want to expose those mocks, right?

Reply:
I have seen examples of both inside internal and root. Not a strong opinion on either of those. So I moved it to internal/